content/en/docs/ops/configuration/mesh/app-health-check/index.md

```diff
@@ -113,7 +113,7 @@ You have two ways to enable Istio to rewrite the liveness HTTP probes.
 **Alternatively**, update the configuration map of Istio sidecar injection:
 
 {{< text bash >}}
-$ kubectl get cm istio-sidecar-injector -n istio-system -o yaml | sed -e 's/"rewriteAppHTTPProbe":false/"rewriteAppHTTPProbe":true/' | kubectl apply -f -
+$ kubectl get cm istio-sidecar-injector -n istio-system -o yaml | sed -e 's/"rewriteAppHTTPProbe":\ false/"rewriteAppHTTPProbe":\ true/' | kubectl apply -f -
```

Comment:
Why the `\`?

Reply:
This is to escape colon delimiter, as it is preceded by space and separate two strings `rewriteAppHTTPProbe` and `false`. Not sure if it works without `\`

Reply:
But the backslash is escaping the following space character, not the colon. Since the whole rule is inside single quotes (`'...'`), and `\ ` has no meaning in sed, as far as I know, so my guess is that the backslash does nothing.

Reply:
Thanks @frankbu , I did try to run the command without `\` and it works perfectly fine and updates `rewriteAppHTTPProbe`.